sdk/src/client/api/block_builder/input_selection/requirement/sender.rs

```diff
@@ -50,7 +50,7 @@ impl InputSelection {
                     for input in self.selected_inputs.iter() {
                         let required_address = input
                             .output
-                            .required_address(self.slot_index, self.protocol_parameters.committable_age_range())?
+                            .required_address(self.creation_slot, self.protocol_parameters.committable_age_range())?
```

Comment:
This needs the commitment slot index instead of the tx creation slot

Reply:
Probably good to also rename the param in `required_address()` to `commitment_slot_index`

Reply:
done